backend/api/Database/Models/IsarStep.cs

```diff
@@ -1,39 +1,69 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 # nullable disable
 namespace Api.Models
 {
-    public class ReportEntry
+    public class IsarStep
     {
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         [Required]
         public string Id { get; set; }
 
         [Required]
-        public virtual Report Report { get; set; }
+        public string IsarStepId { get; set; }
+
+        [Required]
+        public virtual IsarTask Task { get; set; }
 
         public string TagId { get; set; }
 
         [Required]
-        public ReportEntryStatus ReportEntryStatus { get; set; }
+        public IsarStepStatus StepStatus { get; set; }
 
         [Required]
+        public StepType StepType { get; set; }
+
         public InspectionType InspectionType { get; set; }
 
         [Required]
         public DateTimeOffset Time { get; set; }
 
         [MaxLength(128)]
-        [Required]
         public string FileLocation { get; set; }
     }
 
-    public enum ReportEntryStatus
+    public enum IsarStepStatus
+    {
+        Successful,
+        InProgress,
+        NotStarted,
+        Failed,
+        Cancelled
+    }
+
+    public enum StepType
```

Comment:
Not very happy about the "duplication" of `StepType` and `InspectionType` here but not certain what else to do.